columnflow/tasks/framework/mixins.py

```diff
@@ -36,9 +37,9 @@
 
 class ArrayFunctionClassMixin(ConfigTask):
 
-    def array_function_cls_repr(self, array_function) -> str:
+    def array_function_cls_repr(self, array_function: Type[TaskArrayFunction]) -> str:
```

Comment:
```suggestion
    def array_function_cls_repr(self, array_function: str) -> str:
```

Reply:
Added in 0baa5feb (had to remove an import as well)